hack/redis/redis-load-test.yaml

```diff
@@ -0,0 +1,51 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: redis-load-template
+  annotations:
+    description: "load testing redis"
+    iconClass: "icon-redis"
+objects:
+  - apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: redis-load
+    spec:
+      strategy:
+        type: Recreate
+      selector:
+        matchLabels:
+          app: redis-load
+      replicas: 1
+      template:
+        metadata:
+          labels:
+            app: redis-load
+        spec:
+          containers:
+            - env:
+                - name: PORT
+                  value: ${PORT}
+                - name: HOST
+                  value: ${HOST}
+                - name: ENTRIES
+                  value: ${ENTRIES}
+                - name: USERS
+                  value: ${USERS}
+              name: redis-load
+              image: quay.io/croche/redis-load:latest
```

Comment:
Where is the source for it?

Reply:
Happy to move this, or make it better and move it into CRO if you want -> https://github.com/ciaranRoche/redis-load

Reply:
I think it'd be good to pull this in. Mainly for future maintenance and so it doesn't get lost over time.